doc/source/parameters/dem/model_parameters.rst

```diff
@@ -0,0 +1,92 @@
+
+.. note::
+    Lethe-DEM has two linear and non-linear particle-wall models.
+
+.. note::
+    ``euler`` (1st order) and ``velocity-verlet`` (2nd order) are the available integration methods in Lethe-DEM.
+
+.. note::
+    Three rolling resistance models are available in Lethe-DEM: ``no_resistance``, ``constant_resistance``, ``viscous_resistance``.
+
+
+.. code-block:: text
+
+ subsection model parameters
+  # Contact detection method
+  # Choices are constant|dynamic
+  set contact detection method                          = dynamic
+
+  # Depending on the contact detection method, contact search size coefficient (safety factor multiplier for dynamic contact search) or contact detection frequency should be defined for dynamic and constant contact search methods, respectively.
+  set dynamic contact search size coefficient           = 0.9
+  set contact detection frequency                       = 20
+
+  # Load balancing method and its subsequent information
+  set load balance method                               = frequent
+  set load balance frequency                            = 200000
+  set load balance particle weight                      = 10000
+
+  # Particle-particle contact neighborhood size
+  set neighborhood threshold                            = 1.6
+
+  # Particle-particle contact force model
+  # Choices are linear|hertz_mindlin_limit_overlap|hertz_mindlin_limit_force|hertz
+  set particle particle contact force method            = hertz_mindlin_limit_overlap
+
+  # Particle-wall contact force model
+  # Choices are linear|nonlinear
+  set particle wall contact force method                = nonlinear
+
+  # Integration method
+  # Choices are euler|velocity_verlet
+  set integration method                                = velocity_verlet
+
+  # Rolling resistance method
+  # choices are no_resistance|constant_resistance|viscous_resistance
+  set rolling resistance torque method                  = no_resistance
```

Comment:
It would be good to start a theory page where we give the formulas for the contact models, the time integrator, etc. i'll take care of that.

Reply:
This is a very interesting idea.